sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/models/KeyRotationPolicyAction.java

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.security.keyvault.keys.models;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonValue;
+
+/**
+ * Defines the types of key rotation policy actions that can be executed.
+ */
+public enum KeyRotationPolicyAction {
+    ROTATE("rotate"),
```

Comment:
FYI I know the swagger says "rotate" and "notify" but please be aware that while the service accepts both it will return "Rotate" and "Notify" - if that matters for types / deserialization?

Reply:
The serialization/deserialization library uses the `fromString()` method in this enum, which ignores casing, to create an appropriate instance.